erik/calculator/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Sarabun:wght@100&family=Work+Sans:wght@300;360&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="main_div">
+      <div class="dots">
```

Comment:
I'm going to review this as if it were a colleagues code and jot down misc comments for whatever they're worth =). 
It took me a minute to realize what `dots` meant. Obviously these don't do anything at this point but I'd typically try to name something like this a bit more descriptive. Perhaps something like `window-controls` and for each, `window-controls--close` / `window-controls--minimize`, `window-controls--expand`.